packages/ncids-css/packages/usa-button/internal/_external.scss

```diff
@@ -1,24 +1,38 @@
-@import '../../../core/mixins/external_check';
+@use 'uswds-core' as *;
+@use "sass:meta" as meta;
 
-// .no-exit-notification old platform old link disclamer dont show this thing..
-
-%external-link-icon {
-	&:not(.nci-internal, .no-exit-notification) {
-		@include external-check(
-			$theme-internal-domains,
-			$theme-external-link-icon-size,
-			$theme-external-link-icon-color
-		);
+// Mixin to pass for overriding certain external icon styles
+@mixin usa-button-external-icon() {
+	// Display Block to wrap Icon with Text
```

Comment:
I'm not sure if we should be adding external stuff to the button styles since they may not be consistent across components. (or maybe they are, something to double check..) Maybe try keeping the styles specific to CTA Strip and just targeting any `.usa-button` class within `.nci-cta-strip` ?